purgatory.gemspec

```diff
@@ -2,11 +2,11 @@
 # DO NOT EDIT THIS FILE DIRECTLY
 # Instead, edit Jeweler::Tasks in Rakefile, and run 'rake gemspec'
 # -*- encoding: utf-8 -*-
-# stub: purgatory 3.1.0 ruby lib
+# stub: purgatory 3.1.1 ruby lib
 
 Gem::Specification.new do |s|
   s.name = "purgatory"
-  s.version = "3.1.0"
+  s.version = "3.1.1"
```

Comment:
@dobsonken don't forget that you have to change this as well: https://github.com/financeit/purgatory/commit/f4ac0452edfe700453f7bd18a3be6806d07cc3d8